src/Search.elm

```diff
@@ -402,8 +402,13 @@ view path title model viewSuccess outMsg =
                                                 model.from + model.size
                                             )
                                         ++ " of "
-                                        ++ String.fromInt result.hits.total.value
-                                        ++ "."
+                                        ++ (if result.hits.total.value == 10000 then
+                                                "more then 10000. You have reached the maximum of search results we can show."
```

Comment:
```suggestion
                                                "more than 10000. You have reached the maximum of search results we can show."
```

`more then` → `more than`.

I don't like "You have reached the maximum of search results we can show", though not sure what to replace it with.

First off, we need to invite the user to make a more precise search, only being sorry isn't helpful. It's also an implementation detail to say that it's the "maximum we can show".

Maybe something along the lines of "*more than 10000 results, please provide more precise search terms.*"

If we're informal in the app, maybe something like: "*more than 10000 results, that's way too many to be useful. Help us help you and make a more precise search!*"

Reply:
I'll go for the boring formal version.